data/Denmark/Folketing/ep-popolo-v1.0.json

```diff
@@ -573,7 +573,7 @@
       ],
       "sources": [
         {
-          "url": "http://www.thedanishparliament.dk/Members/aleqa-hammond.aspx"
+          "url": "http://www.thedanishparliament.dk/Members/smalha.aspx"
```

Comment:
This doesn't seem to be a valid URL. http://www.thedanishparliament.dk/Members/aleqa-hammond.aspx still seems to be the correct one.